src/analytics.ts

```diff
@@ -22,6 +22,7 @@ export default class Analytics {
     ).replace(/^FUBAR/, 'UA');
 
     constructor(context: vscode.ExtensionContext) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         this.extension = vscode.extensions.getExtension(
             'betterthantomorrow.calva'
         )!;
```

Comment:
In general it's better to fix types than do a non-null assertion, but in this case since it's something the type system can't figure out we can just do an assert instead..